### ISSUE TITLE
Consider having colour.firefox.com also resolve to FirefoxColor

### ISSUE THREAD

**Issue by chutten** [mozilla/FirefoxColor#345]
Right now it fails DNS. I figure it could be a nice nod for the majority of English speakers who spell it that way.

**Comment by nt1m** [comment]
Did you mean color ?

**Comment by chutten** [comment]
color.firefox.com works just fine. I was hoping we could have colour.firefox.com resolve in addition.

**Comment by lmorchard** [comment]
FWIW, if we did, it would have to be a redirect. The add-on is specifically tied to the color.firefox.com origin for its permissions

**Comment by chutten** [comment]
Fair enough. This is only a "wouldn't it be nice"-level enhancement request.

**Comment by johngruen** [comment]
@jbuck what do you think? we'd have to do a redirect?

**Comment by jbuck** [comment]
@johngruen yeah, we'd need to setup a redirect

**Comment by johngruen** [comment]
@jbuck if you have time?
